src/azul/service/index_query_service.py

```diff
@@ -61,15 +68,15 @@ def get_data(self,
                  file_url_func: FileUrlFunc,
                  item_id: Optional[str] = None,
                  filters: Optional[str] = None,
-                 source_ids: Optional[Set[str]] = None,
+                 source_filter: Optional[SourceFilter] = None,
```

Comment:
Doesn't need to be optional. I see only one call and the parameter is always passed and never None. We're always either filtering or indicating. So

```suggestion
                 source_ids: Set[str],
                 filter_sources: bool
```

And force all arguments to be passed as keyword arguments by adding `*` after `self`.